terraform_module/outputs.tf

```diff
@@ -0,0 +1,38 @@
+output "subscription_id" {
```

Comment:
Why this new naming? modules/* -> terraform_module? curious ...

Reply:
To enable parallelism and isolation locally for running these tests I wanted to move all terraform config in to a tempDir in each test. To this end I need a folder that continues the root module for Terraform to avoid copying useless stuff in there during the test. This folder `terraform_module` represents the root module. Alternatives I can see are

1) Rename it so it's clearer, e.g. `root_module` or `jx_module`
2) Move it as a sibling folder of the other lower level modules (but this doesn't make it clear which module is the root/executable one)

Reply:
Renamed to `jx_module` for now